internal/controller/scheduler/scheduler.go

```diff
@@ -168,8 +169,15 @@ func (w *worker) Handle(ctx context.Context, job *api.AgentScheduledJob) error {
 			return fmt.Errorf("%w: %w", model.ErrDuplicateJob, err)
 
 		case kerrors.IsInvalid(err):
+			out := ""
+			outBytes, errYaml := yaml.Marshal(kjob)
+			if errYaml != nil {
+				out = fmt.Sprintf("%v", errYaml)
```

Comment:
Error messages start lower-case to make them easier to incorporate into longer sentences. This would be the place to provide that context for human consumption:

```suggestion
				out = fmt.Sprintf("Couldn't marshal the Job into YAML: %v", marshalErr)
```